### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by tkz96**
This PR is for:

1. Writing test cases for the entire Math Magicians application
2. Test whether the application is working as intended or not.

### CODE REVIEW

File: src/components/__test__/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Calculator from '../Calculator';
+
+describe('Test for the Calculator component', () => {
+    it('Should render the Calculator', () => {
+      const tree = renderer.create(<Calculator />);
+      expect(tree).toMatchSnapshot();
+    });
+});
```
Comment (medaminedev66) [inline src/components/__test__/Calculator.test.js]: - [x] Following the project requirements, you should use also React Testing Library to simulate user interaction. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.

File: src/pages/__test__/Quote.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Quote from '../Quote';
+
+describe('Test for the Quote page', () => {
+    it('Should render the Quote page', () => {
+      const tree = renderer.create(<Quote />);
+      expect(tree).toMatchSnapshot();
+    });
+});
```
Comment (medaminedev66) [inline src/pages/__test__/Quote.test.js]: - [x] As I stated in the previous comment, you should also use React Testing Library to simulate user interaction. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.
- Kindly use React Testing Library to stimulate user interaction for all the components.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ibr5500 and @tkz96  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- There are no linter errors ✔️
- The tests run succussfully✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Hi @medaminedev66, 
We have made the required changes.
Thank you for the code review.

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Hi @DeliceLydia, 

Thank you for the review.

**Pull request merged**